pkg/services/accesscontrol/roles.go

```diff
@@ -0,0 +1,115 @@
+package accesscontrol
+
+import "github.com/grafana/grafana/pkg/models"
+
+var PredefinedRoles = map[string]RoleDTO{
+	usersSelfRead: {
+		Name:    usersSelfRead,
+		Version: 1,
+		Permissions: []Permission{
+			{
+				Permission: "users:read",
+				Scope:      ScopeUsersSelf,
+			},
+			{
+				Permission: "users.tokens:list",
+				Scope:      ScopeUsersSelf,
+			},
+			{
+				Permission: "users.teams:read",
+				Scope:      ScopeUsersSelf,
+			},
+		},
+	},
+	usersAdminRead: {
+		Name:    usersAdminRead,
+		Version: 1,
+		Permissions: []Permission{
+			{
+				Permission: ActionUsersRead,
+				Scope:      ScopeUsersAll,
+			},
+			{
+				Permission: ActionUsersTeamRead,
+				Scope:      ScopeUsersAll,
+			},
+			{
+				Permission: ActionUsersAuthTokenList,
+				Scope:      ScopeUsersAll,
+			},
+			{
+				Permission: ActionUsersQuotasList,
+				Scope:      ScopeUsersAll,
+			},
+		},
+	},
+	usersAdminEdit: {
+		Name:    usersAdminEdit,
+		Version: 1,
+		Permissions: []Permission{
```

Comment:
It seems this one is missing users read and teams read? 
```
{
				Permission: ActionUsersRead,
				Scope:      ScopeUsersAll,
			},
			{
				Permission: ActionUsersTeamRead,
				Scope:      ScopeUsersAll,
			},
```

Reply:
🤔 Oh yeah, we talked about making it so that edit should inherit read 🤦. Ugh. I don't want to duplicate this everywhere, it's too fragile to manually inherit permissions that way. Will have to rebuild some things to accomodate for that.